Block/AjaxBlockService.php

```diff
@@ -0,0 +1,114 @@
+use Sonata\BlockBundle\Model\BlockInterface;
+use Sonata\CoreBundle\Model\Metadata;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\OptionsResolver\OptionsResolverInterface;
+
+/**
+ * Class AjaxBlockService
+ *
+ * @package Sonata\DashboardBundle\Block
+ * @author  Quentin Somazzi <qsomazzi@ekino.com>
+ */
+class AjaxBlockService extends BaseBlockService
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function execute(BlockContextInterface $blockContext, Response $response = null)
+    {
+        return $this->renderResponse($blockContext->getTemplate(), array(
+            'block_context' => $blockContext,
+            'block'         => $blockContext->getBlock(),
+            'settings'      => $blockContext->getSettings(),
+        ), $response);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function buildEditForm(FormMapper $form, BlockInterface $block)
+    {
+        $form->add('settings', 'sonata_type_immutable_array', array(
+            'label' => false,
+            'keys'  => array(
+                array('title', 'text', array()),
+                array('class', 'text', array()),
+                array('url', 'text', array()),
+                array('icon', 'text', array()),
+                array('color', 'choice', array('choices' => $this->getColorChoices())),
+            ),
+        ));
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function setDefaultSettings(OptionsResolverInterface $resolver)
+    {
+        $resolver->setDefaults(array(
+            'template' => 'SonataDashboardBundle:Block:ajax.html.twig',
+            'url'      => null,
+            'color'    => null,
+            'icon'     => 'fa fa-dashboard',
+            'title'    => '',
+            'class'    => 'col-md-3 col-sm-6 col-xs-12',
+        ));
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getName()
+    {
+        return 'Ajax (core)';
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getBlockMetadata($code = null)
+    {
+        return new Metadata($this->getName(), ($code ?: $this->getName()), false, "SonataDashboardBundle", array('class' => 'fa fa-dashboard'));
+    }
+
+    /**
+     * @return array Available list of background colors
+     */
+    private function getColorChoices()
+    {
+        return array(
+            'red'        => 'Red',
+            'yellow'     => 'Yellow',
+            'aqua'       => 'Aqua',
+            'blue'       => 'Blue',
+            'light-blue' => 'Light blue',
+            'green'      => 'Green',
+            'navy'       => 'Navy',
+            'teal'       => 'Teal',
+            'olive'      => 'Olive',
+            'lime'       => 'Lime',
+            'orange'     => 'Orange',
+            'fuchsia'    => 'Fuchsia',
+            'purple'     => 'Purple',
+            'maroon'     => 'Maroon',
+            'black'      => 'Black',
+            'gray'       => 'Gray',
```

Comment:
@Soullivaneuh a trait like this one will do the trick ? 
https://github.com/qsomazzi/portal.v2/blob/master/src/Portal/Component/Core/Color.php
What do you think ?

Waiting for @rande opinion to see where we put it.